wgpu-hal/src/dx12/command.rs

```diff
@@ -732,46 +738,50 @@ impl crate::CommandEncoder for super::CommandEncoder {
         }
 
         if let Some(ref ds) = desc.depth_stencil_attachment {
-            let mut flags = d3d12::ClearFlags::empty();
+            let mut flags = Direct3D12::D3D12_CLEAR_FLAGS::default();
             let aspects = ds.target.view.aspects;
             if !ds.depth_ops.contains(crate::AttachmentOps::LOAD)
                 && aspects.contains(crate::FormatAspects::DEPTH)
             {
-                flags |= d3d12::ClearFlags::DEPTH;
+                flags |= Direct3D12::D3D12_CLEAR_FLAG_DEPTH;
             }
             if !ds.stencil_ops.contains(crate::AttachmentOps::LOAD)
                 && aspects.contains(crate::FormatAspects::STENCIL)
             {
-                flags |= d3d12::ClearFlags::STENCIL;
+                flags |= Direct3D12::D3D12_CLEAR_FLAG_STENCIL;
             }
 
-            if !ds_view.is_null() && !flags.is_empty() {
-                list.clear_depth_stencil_view(
-                    unsafe { *ds_view },
-                    flags,
-                    ds.clear_value.0,
-                    ds.clear_value.1 as u8,
-                    &[],
-                );
+            if let Some(ds_view) = ds_view {
+                if flags != Direct3D12::D3D12_CLEAR_FLAGS::default() {
+                    unsafe {
+                        list.ClearDepthStencilView(
+                            ds_view,
+                            flags,
+                            ds.clear_value.0,
+                            ds.clear_value.1 as u8,
+                            &[],
```

Comment:
This one used to be `ptr::null()` too, but there's no `None` mapping in Direct3D12 here because someone only reported this issue (and hence fixed it) for `clearRenderTargetView()` 🤦🤦.

https://github.com/microsoft/win32metadata/pull/1971